pybricksdev/ble.py

```diff
@@ -29,14 +29,15 @@ async def find_device(name: str, timeout: float = 5) -> BLEDevice:
         asyncio.TimeoutError:
             Device was not found within the timeout.
     """
-    print("Searching for {0}".format(name))
+    print("Searching for {0}".format(device_or_address))
```

Comment:
Since we need to make some other changes anyway, we could change this to:

```suggestion
    print(f"Searching for {device_or_address}")
```